.github/scripts/printPodspec.rb

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ruby
+
+# This file is a lightweight port of the `pod ipc spec` command.
+# It was built from scratch to imports some 3rd party functions before reading podspecs
+
+require 'cocoapods'
+require 'json'
+
+# Require 3rd party functions needed to parse podspecs. This code is copied from ios/Podfile
+def node_require(script)
```

Comment:
_alternate solution:_ DRY this up between this file and the Podfile. Didn't go for this because it will make RN upgrades harder. The more similar our Podfile is to the standard template, the easier.

_alternate solution:_ import the `Podfile` directly with `load '../../ios/Podfile'`. I like this, but I was getting errors like `undefined method 'platform'`, and it wasn't obvious all the things I would have to require in order to make this script understand how to parse a Podfile. I would've though just `require 'cocoapods'` would've been enough, but I guess not.

Reply:
Thanks for sharing, current solution seems fine for now.